app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -291,6 +291,8 @@ def check_connect_api(opts = {})
       verify_ssl = opts[:verify_ssl] || 1
       ca_certs = opts[:ca_certs]
 
+      return true if server.blank?
```

Comment:
This now matches the logic that the metrics credentials check does: https://github.com/ManageIQ/manageiq-providers-ovirt/pull/551/files#diff-e275a9678ece17256de11e64463619843136b35ff28ec827c8f571d0c418f5feR348

Reply:
this link does not seem to take me anywhere

Reply:
Weird, it is line 348 in this file